web/ui/module/codemirror-promql/src/lang-promql/grammar/promql.grammar

```diff
@@ -16,7 +16,8 @@
 
 @precedence {
   pow @right,
-  mul @left,
+  mul @left
+  atan2 @left,
```

Comment:
I think this is the right order.

Reply:
I think this line isn't needed, the precedence level `mul` is used for all the operators of that precedence level (`*`, `/`, `%`, `atan2`).

Reply:
could be cool to add a test in the file `expression.txt` to verify it works